numbat/src/ffi.rs

```diff
@@ -903,3 +912,11 @@ fn from_unixtime(args: &[Value]) -> Result<Value> {
 
     Ok(Value::DateTime(dt))
 }
+
+fn random(args: &[Value]) -> Result<Value> {
+    assert!(args.len() == 0);
+
+    let output = fastrand::f64();
```

Comment:
We could use the `rand` crate instead (it's already a transitive dependency of Numbat).

Reply:
I've changed it to `rand` and removed the test.